CommunityToolShedMvc/Models/Tool.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace CommunityToolShedMvc.Models
+{
+    public class Tool
+    {
+        public int ID { get; set; }
+
+        [Required]
+        public string Name { get; set; }
+
+        [Required]
+        public int OwnerID { get; set; }
```

Comment:
The `Required` attribute isn't necessary to use on non-nullable value type properties.